sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/ClientConstants.java

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.servicebus.implementation;
+
+import java.time.Duration;
+
+/**
+ *  Constants defined for service bus.
+ */
+public class ClientConstants {
```

Comment:
This should have a more descriptive name

Reply:
I have updated documentation `Declares at which time the message should appear on the Service Bus Queue.`

Reply:
`ServiceBusClientConstants`    Does this make sense ?

Reply:
Removed this class for now.